app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def _is_enough_power(self, car: Car) -> bool:
+        return self.clean_power > car.clean_mark
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.distance_from_city_center == 0:
+            self.distance_from_city_center = 1
```

Comment:
The current approach to handle division by zero by setting `distance_from_city_center` to 1 is a workaround. Consider a more robust solution, such as raising an exception or using a default value that logically fits your application's context.